### PR TITLE
[FIX] Iterating over Profile list selector

### LINKED ISSUE #39: Iterate over the list of profileListSelectors
**Is your feature request related to a problem? Please describe.**
Generalize this section of code to use a loop instead of hardcoding array indexes. This would be useful if we want to add any more profileListSelectors in the future. Moreover, generalizing it would help in code clarity (Add comments to explain the approach in the code).

**https://github.com/ayushjainrksh/conactivity/blob/83c3c40fdbe8d88679eca637f018fabacf14bdf2/scrape.js#L110-L114**

**Additional context**
To understand what this piece of code does, go through #30

### PR DESCRIPTION
**Pull request by amandesai01**
## Description

Iterating over list of `profileListSelectors` and finding first selector having data.

Fixes #39 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by ayushjainrksh** [comment]
Thanks for the PR! I'll have a look very soon.

**Comment by amandesai01** [comment]
Is everything correct? If yes, can you approve it, or you can suggest me changes. Anything will work.
FYI: When locally tested, everything works just as it worked before this PR. So, no issue there.

**Comment by ayushjainrksh** [comment]
@amandesai01 I'm sorry for delaying the merge. The PR looks fine to me, I need to test it locally. No actions required as of now.

File: scrape.js [hunk 1]
```diff
@@ -107,11 +107,24 @@ const scrapeLinkedIn = async (data) => {
           ".search-result__info .search-result__result-link",
           ".reusable-search__entity-results-list .entity-result__title-text a",
         ];
-        const profileListNodes =
-          (document.querySelectorAll(profileListSelectors[0]).length &&
-            document.querySelectorAll(profileListSelectors[0])) ||
-          (document.querySelectorAll(profileListSelectors[1]).length &&
-            document.querySelectorAll(profileListSelectors[1]));
+        let profileListNodes = undefined;
```
Comment (ayushjainrksh) [inline scrape.js:110]: `null` would be a better option here than `undefined` because it's safer to use and has a type `object` rather than `undefined`.

**Review by ayushjainrksh — APPROVED** [review]

**Comment by ayushjainrksh** [comment]
@amandesai01  This looks good. Thanks for the PR!

**Pull request merged**